docs/faq/proof-of-stake/stake-pools.md

```diff
@@ -4,11 +4,10 @@
 
 #### **1. Are there any other benefits to a stake pool other than not needing to run a full node and keeping your wallet unlocked? For example, will it have a better chance of winning or anything like that?**
 
-Well first keep in mind that wallet (`dcrwallet`) != core network (`dcrd`). In order to stake you need to have `dcrwallet` online and unlocked 24/7.
+Stake pools will usually implement multi-wallet redundancy by having many wallets physically distributed around the globe. This means there's less chance of a vote
```

Comment:
They should be; I think that was part of the RFP. I know Bravo is since I helped set it up. I'm pretty sure the rest are as well.